com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -23,19 +23,25 @@ internal class RpcQueueProcessor
         // Batcher object used to manage the RPC batching on the send side
         private readonly RpcBatcher m_RpcBatcher = new RpcBatcher();
         private const int k_BatchThreshold = 512;
+        private NetworkingManager m_NetManager;
```

Comment:
So it appears we have an NM instance in this class so that:
- we can get an NM-specific RPC Queue Container, which I have some doubts on is the right abstraction
- we can call NM:InvokeRPC (and, in turn, InvokeRPC is only in NM because it needs to find the right spawn manager, and maybe there's a cleaner way to do that)
-  we can call NM:MessageSender.Send()